browbeat.yml

```diff
@@ -27,6 +27,14 @@
           ansible-playbook -i hosts.yml install/browbeat.yml
       args:
           chdir: "{{ browbeat_dir }}/ansible"
+      when: tls_everywhere != true
+    - name: install browbeat
+      shell: |
+          source /home/stack/.bash_profile
```

Comment:
why do we need to separate tasks for installing browbeat?

Reply:
`REQUESTS_CA_BUNDLE` is added to .bash_profile for the python requests module to recognize the IdM server as a valid certificate authority.
The .bash_profile file has to be sourced before Browbeat installation as it requires `REQUESTS_CA_BUNDLE` to be set as well.

Reply:
can we have one single task to install browbeat.